lib/content-services/src/lib/tag/tags-creator/tags-creator.component.scss

```diff
@@ -38,7 +38,8 @@ adf-tags-creator {
         display: inline-block;
         padding-right: 12px;
         overflow: auto;
-        max-height: 7vh;
+        width: 75%;
```

Comment:
I don't think this should be the solution for the error mentioned in the Jira, this should be fixed in ACA as the new layout caused the error. The problem has nothing to do with T&C which has been working in ACA and control-center before layout changes. Also this might break dialog for creating tags in control-center. @AleksanderSklorz do you agree?

Reply:
Yes I completely agree with you @MichalKinas

Reply:
We have demoed tags and categories earlier in old UI and it was working fine so it has nothing related with Tags and Categories.

Reply:
I looked up again, found a class missing in ACA which had happened during layout changes, I have added the class, raised another PR for it. But still suggesting to remove max-height from adf-tags-creator as it is causing a second scroll when the tag name is large. You can remove it later. https://github.com/Alfresco/alfresco-content-app/pull/3150
![image](https://user-images.githubusercontent.com/112551534/234551863-f8b08a4b-ceb2-413f-9d6a-64ff29434fce.png)

Reply:
I think this scrollbar is to avoid situation that somebody will type really very long tag name so user would need to scroll down to see existing tags. Not sure if for categories we have scrollbar here @MichalKinas? We can remove scrollbar if the same is for categories, that's fine for me. Please also check if that does not break tags modal in control center. 
In case we want to remove that scrollbar then only change in that PR would be removing max-height I guess

Reply:
I think Categories doesn't have it since it would be really rare case so removing max-height is fine for me, but for sure we shouldn't modify the width and check if nothing breaks in control-center